### ISSUE TITLE
[AutoComplete][v1] Options popover is not shown again after selection unless a onBlur occurs

### ISSUE THREAD

**Issue by adamcarchibald** [Shopify/polaris#1265]
# Issue summary
If a user finds and selects a match strictly with keyboard inputs and then begins erasing the input to find another match, the options that match are not displayed until you click outside of the text field.

## Expected behaviour
If options are available and the Text Field onChange occurs, the options should be displayed again in a popover.

## Actual behaviour

![](https://screenshot.click/2019-03-28_13-25-49.gif)

Can be replicated at https://polaris.shopify.com/components/forms/autocomplete

**Comment by ry5n** [comment]
Thanks for the report. I can replicate.

**Comment by dleroux** [comment]
Autocomplete is currently being reworked as a part of the [AutoComplete component project](https://github.com/Shopify/polaris-react/projects/3). Once we address this problem in the reworked component, we’ll close this issue.

**Comment by yurm04** [comment]
I believe this has been fixed ✅
